stylish-haskell.cabal

```diff
@@ -128,8 +132,8 @@ Test-suite stylish-haskell-tests
     aeson            >= 0.6    && < 1.5,
     base             >= 4.8    && < 5,
     bytestring       >= 0.9    && < 0.11,
+    Cabal            >= 2.4    && < 2.5,
```

Comment:
I am very hazy on the concept of version bounds, could you quickly explain the motivation for [2.4, 2.5) interval?

Reply:
So what we're doing is trying to specify bounds to prevent tools from trying to build stylish-haskell with the wrong version of Cabal.  You and both tested this with 2.4, so that's fine.  Any further release in the 2.4.X range _must_ be compatible with 2.4.

However, Cabal 2.5 is not out yet, and since that bumps a major version, it may break the part of the API we're using. Therefore, I prefer to error on the side of caution here.  If 2.5 comes out and it's compatible, it's a quick fix anyway.

Reply:
Thanks for clarifying on this @jaspervdj !